### PR TITLE
Expose github.Storer and use it in constructor

### PR DESCRIPTION
**Pull request by smacker**
I simplified the constructor a little while working on unified metadata PR.

---

* [ ] I have updated the CHANGELOG file according to the conventions in [keepachangelog.com](https://keepachangelog.com)
* [x] This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

File: examples/cmd/main.go [hunk 1]
```diff
@@ -233,13 +234,13 @@ type bodyFunc = func(logger log.Logger, downloadersPool *DownloadersPool) error
...
 func (c *DownloaderCmd) ExecuteBody(logger log.Logger, fn bodyFunc) error {
 	ctx := context.Background()
-	var db *sql.DB
+	var storer github.Storer
 	if c.DB == "" {
 		log.Infof("using stdout to save the data")
-		db = nil
+		storer = &store.Stdout{}
 	} else {
 		var err error
```
Comment (dpordomingo) [inline examples/cmd/main.go]: I think we no longer need this line, do we?

Comment (dpordomingo) [reply examples/cmd/main.go]: And if I'm not wrong, we could also Close `db` in `defer`, no matter if we got an error or not (e.g. If I understood it properly, we're not closing it if `database.Migrate(c.DB)` succeeded)

**Review by dpordomingo — APPROVED** [review]
wow! I had the very same in a branch which I forgot to PR after opening the PR handing api limits!
so it LveryGTM

also, since you're refactoring this, you could maybe consider these two suggestions, which are close to your editions.

**Review by se7entyse7en — APPROVED** [review]

File: examples/cmd/main.go (hunk 1 above)
Comment (smacker) [reply examples/cmd/main.go]: yes, thanks

Comment (smacker) [reply examples/cmd/main.go]: fixed

**Pull request merged**